pkg/remote/aws/init_test.go

```diff
@@ -1,6 +1,8 @@
 package aws
 
-import "github.com/cloudskiff/driftctl/pkg/terraform"
+import (
+	"github.com/cloudskiff/driftctl/pkg/terraform"
+)
```

Comment:
Is this your linter that changed these lines ?

Reply:
might be my goland config I'll check